lib/repo.rb

```diff
@@ -60,6 +60,6 @@ def source
   end
 
   def url_for_file(file)
-    File.join(HOST, @org, @name, @branch, file)
+    File.join(HOST, @org, @name, 'raw', @branch, file)
```

Comment:
Why this literal `raw` here?

Reply:
That's just what a GitLab URl looks like -- github has raw.github.com, GitLab has a special route of the form 'org/name/raw/path/to/file'.